src/odemis/driver/andorcam2.py

```diff
@@ -2073,6 +2074,7 @@ def _update_settings(self):
         self._metadata[model.MD_EXP_TIME] = exposure
         readout = im_res[0] * im_res[1] * self._metadata[model.MD_READOUT_TIME] # s
         # accumulate should be approximately same as exposure + readout => play safe
+        # TODO: check if that includes the shutter time (can be 2 x 50 ms per frame!)
```

Comment:
Is this TODO for later or for this PR?

Reply:
This one is to remember doing it that, later.